core/arch/arm/kernel/kern.ld.S

```diff
@@ -394,6 +394,7 @@ SECTIONS
 	__init_size = __data_end - CFG_TEE_LOAD_ADDR;
 	__init_mem_usage = __end - CFG_TEE_LOAD_ADDR;
 #endif
+	ASSERT(. <= (CFG_TEE_RAM_START + CFG_TEE_RAM_VA_SIZE), "the CFG_TEE_RAM_VA_SIZE is not enough!")
```

Comment:
minor: split over 2 lines for 80char/line constraint.

This adds an explicit build error trace "... is not enough!" to an error case that is IMO  already implicit in the next line. Hence I think this assert is ok, but I let @jenswi-linaro and @jforissier consider whether it is uselessly redundant or not.